big.js

```diff
@@ -13,6 +13,43 @@ window.onload = function() {
   }
   if (!s.length) return;
   var big = window.big = { current: 0, forward: fwd, reverse: rev, go: go, length: s.length };
+  var audioCandidates = document.getElementsByTagName('audio');
+  for(var ai = 0; ai < audioCandidates.length; ai++)
+    if (audioCandidates[ai].textTracks.length === 1 && audioCandidates[ai].textTracks[0].cues.length > 0) {
+      big.audio = audioCandidates[ai];
+      var playControl = document.createElement('img');
+      playControl.style = 'width:30px;height:30px;';
+      playControl.id = 'audio-control';
+      playControl.onclick = function(e) {
+        if (big.audio.paused) {
+          if (big.current === 0)
+            big.fwd();
+          else
+            big.audio.play();
+        }
+        else
+          big.audio.pause();
+        e.stopPropagation();
+      }
+      document.getElementsByTagName('body')[0].appendChild(playControl);
+      window.setInterval(function() {
+        if (!big.audio.paused) {
+          document.getElementById('audio-control').src = 'data:image/svg+xml;charset=utf-8,%3Csvg%20width%3D%221792%22%20height%3D%221792%22%20viewBox%3D%220%200%201792%201792%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Cpath%20d%3D%22M832%201184v-576q0-14-9-23t-23-9h-256q-14%200-23%209t-9%2023v576q0%2014%209%2023t23%209h256q14%200%2023-9t9-23zm448%200v-576q0-14-9-23t-23-9h-256q-14%200-23%209t-9%2023v576q0%2014%209%2023t23%209h256q14%200%2023-9t9-23zm384-288q0%20209-103%20385.5t-279.5%20279.5-385.5%20103-385.5-103-279.5-279.5-103-385.5%20103-385.5%20279.5-279.5%20385.5-103%20385.5%20103%20279.5%20279.5%20103%20385.5z%22%20fill%3D%22%23aaa%22%2F%3E%3C%2Fsvg%3E';
+          for(var ci = 0; ci < big.audio.textTracks[0].cues.length; ci++)
```

Comment:
I see a documented [activeCues](https://developer.mozilla.org/en-US/docs/Web/API/TextTrack) property that appears to express what this loop finds.